package.json

```diff
@@ -28,7 +28,7 @@
     "dist:all": "cross-env GENERATE_SOURCEMAP=false npm run build && electron-builder -mwl -c.extraMetadata.main=build/electron.js",
     "publish": "yarn dist --publish always",
     "publish:all": "yarn dist:all --publish always",
-    "test": "node scripts/test.js --maxWorkers=2 --env=node",
+    "test": "node scripts/test.js --maxWorkers=2 --env=node --verbose=false",
```

Comment:
Note: I'll remove verbose later today. So we're getting the default output. With `verbose=false` console.logs are visible in the console but not every test step.